Digital-Peak-Incubator/tpl_tauristar
# Initial Dynamic Module Position Release

Pull Request for Issue # .

#### Summary of Changes
Utilizes the new Subform Field for J3.6 to allow dynamic Module Positions
#### Testing Instructions
Go to the Template Stile Options and create unter the Tab "Modules" some Module Positions.
Then create some Modules with the new Positions.
Test if the modules appear.
#### Documentation Changes Required
